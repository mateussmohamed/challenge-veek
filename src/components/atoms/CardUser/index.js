import styled from 'styled-components'

const CardUser = styled.div`
  display: flex;
  align-items: center;
`

export default CardUser
