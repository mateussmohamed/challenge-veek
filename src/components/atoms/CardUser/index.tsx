import styled from 'styled-components'

export const CardUser = styled.div`
  display: flex;
  align-items: center;
`
