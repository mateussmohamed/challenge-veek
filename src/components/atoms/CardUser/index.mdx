---
name: CardUser
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardUser from './index'

# CardUser
<PropsTable of={CardUser} />

<Playground>
  <CardUser />
</Playground>
