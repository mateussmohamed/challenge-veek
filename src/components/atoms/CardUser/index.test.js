import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardUser from './index'

describe('[<CardUser />]', () => {
  const wrap = (props = {}) => shallow(<CardUser {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
