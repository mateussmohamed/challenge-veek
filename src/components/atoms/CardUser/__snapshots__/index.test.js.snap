// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardUser />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardUser-thd5qk-0",
        "isStatic": true,
        "rules": Array [
          "display:flex;align-items:center;",
        ],
      },
      "displayName": "CardUser",
      "render": [Function],
      "styledComponentId": "CardUser-thd5qk-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
